SKILL.tsv

```diff
@@ -921,114 +921,114 @@ SKILL_20150317_000920	Thrust
 SKILL_20150317_000921	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Stab and push away enemy.
 SKILL_20150317_000922	Attack #{SkillAtkAdd}#{nl}Splash #{SkillSR}#
 SKILL_20150317_000923	Bash
-SKILL_20150317_000924	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Swing weapon widely to damage target.
+SKILL_20150317_000924	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Inflict damage with your weapon to a target with a wide swing.
```

Comment:
Inflict conveys that you are performing the action.
"[You will] inflict damage with your weapon to a target with a wide swing."
Inflicts conveys that the skill is performing the action.
"[This skill] inflicts damage with your weapon to a target with a wide swing."

The same is true with the other examples.  Either is correct, but the first sentence could still be formed from the perspective of the skill:
"[Using this skill,] [you will] inflict damage with your weapon to a target with a wide swing."